polylogue/cli/app.py

```
@@ -307,31 +350,31 @@ def run_prune_cli(args: argparse.Namespace, env: CommandEnv) -> None:
-            try:
-                if path.is_dir():
-                    shutil.rmtree(path)
-                else:
-                    path.unlink()
-                removed_here += 1
-            except Exception as exc:
-                ui.console.print(f"[red]Failed to remove {path}: {exc}")
-        if removed_here:
-            ui.console.print(f"[green]Pruned {removed_here} legacy path(s) in {root}")
-            total_removed += removed_here
+            all_legacy.append((root, path))
```

Ruff (0.15.9)

[warning] Use `list.extend` to create a transformed list

(PERF401)